sourcegraph/learn
# Batch update dependencies

### What should this PR do?
Resolves [DEVED-232](https://sourcegraph.atlassian.net/browse/DEVED-232) by updating dependencies.

### Why are we making this change?
- Need to keep repo up-to-date.

### What are the acceptance criteria? 
- Everything should work as it does in production.

### How should this PR be tested?
- Check out the branch, remove your `node_modules`, and install dependencies with `npm ci`.
- Check that everything looks like it does in production.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
